### PR TITLE
pytest.ini: addopts, testpaths

### PR DESCRIPTION
**Pull request by blueyed**

### CODE REVIEW

File: pytest.ini [hunk 1]
```diff
@@ -1,5 +1,4 @@
 [pytest]
-addopts = -ra
```
Comment (blueyed) [inline pytest.ini]: ?
I find this a useful default.

**Comment by tarpas** [comment]
Daniel, it's really a matter of taste and mine leans towards the default
default, which I mentioned to @Matlino

**Comment by blueyed** [comment]
Yeah, I've heard you are not selecting tests manually etc - which the summary makes easy.. ;)

I often double click failed nodeids from there to re-run the test.

IIRC the default is to have no summary at all, no?

**Comment by tarpas** [comment]
There is this summary ;)
```============== 161 passed, 3 skipped, 3 xfailed in 11.84 seconds ===============```

But I might add -rfE soon, thanks for your arguments :).

**Comment by blueyed** [comment]
yeah, having nodeids for failed tests there in a compact list is very useful.
With pytest 4.5 you also get the error there - very useful to see common patterns.

**Pull request merged**